freespace2/freespace.cpp

```diff
@@ -1360,7 +1367,13 @@ bool game_start_mission()
 
 	game_busy( NOX("** starting mission_load() **") );
 	load_mission_load = (uint) time(nullptr);
-	if ( !mission_load(Game_current_mission_filename) ) {
+	bool load_success = mission_load(Game_current_mission_filename);
+	load_mission_load = (uint)(time(nullptr) - load_mission_load);
```

Comment:
Er, what's the reason for moving this up here?

Reply:
Because if the mission load is unsuccessful, the variable will not be updated.

It's for the same reason I moved `stop_parse` - if the mission load is unsuccessful, the memory wouldn't be freed until, presumably, the next time a mission needed to be parsed.

Reply:
I'm unclear on what `load_mission_load` is even for. I see no uses of it after this line.

Reply:
I think it could be for timing or profiling.  I left it in in case it could be usable in the future.  There are other apparently orphaned variables too; maybe this could be addressed in a future PR.

Reply:
Yes, definitely agree that leaving it be was the right thing to do, for the reasons you mentioned. :+1: